containers/buildAlpine7.docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM alpine:3.17
+MAINTAINER Max Neunhoeffer <hackers@arangodb.com>
+
+ARG ARCH "x86_64"
+
+RUN apk update && \
+    apk add groff bison flex make cmake ccache python3 git linux-headers vim boost-dev ctags mandoc gdb fish openssh db-dev file libltdl zlib-dev curl coreutils texinfo gcovr py-setuptools valgrind libtool nodejs npm && apk add g++ prometheus bash
```

Comment:
please add py3-psutil as well here

Reply:
Will be added through https://github.com/arangodb/oskar/pull/491.